deploy/crds/aws_v1alpha1_awsfederatedrole_networkmgmt_cr.yaml

```diff
@@ -34,7 +34,7 @@ spec:
           - "ec2:DescribeTransitGatewayAttachments"
           - "ec2:DescribeTransitGatewayVpcAttachments"
           - "ec2:DescribeTransitGateways"
-          - "ec2:DescribeVPCEndpoint"
```

Comment:
Do we have our own validator?  If not, then it's because AWS's validator only validates that it's valid JSON: https://docs.aws.amazon.com/IAM/latest/UserGuide/access_policies_policy-validator.html

Reply:
@iamkirkbater We actually create the policy, so in theory this would fail. https://github.com/openshift/aws-account-operator/blob/master/pkg/controller/awsfederatedrole/awsfederatedrole_controller.go#L167

Reply:
I guess the point I'm trying to make is that both we and AWS are not validating individual actions against a master list of available actions.  AWS's validation is limited to making sure there is an Actions object that is a type of list of strings, not that the individual Actions therein are correct and match up to an actual permission.

Reply:
Yep, confirmed.  Created a policy, added good `ec2:` list permissions, and then went back in and edited the json and added an action for `ec2:GarbageCollection` and it went through with no issues.

Reply:
@iamkirkbater Can you create the policy with the fake perms?

Reply:
Yep, was also able to create a policy with the following JSON:

```{
    "Version": "2012-10-17",
    "Statement": [
        {
            "Effect": "Allow",
            "Action": [
                "ec2:Describe*",
                "ec2:Garbage"
            ],
            "Resource": "*"
        }
    ]
}```

Reply:
So this is an AWS issue then. Is there even a way we can verify this?

Reply:
Validating each action listed is a valid one?  If there's not anything in the AWS API that does it then I'm going to say it's probably not worth the effort.

It would be easier to write some kind of integration test that creates the policy, assumes it, and then tries to do everything that role/policy combo needs to do.